pepper-apis/dsm-core/src/main/resources/application.conf

```diff
@@ -168,16 +168,6 @@ portal {
 		and deactivated_date is null
     """
 
-    insertKitTrackingRequest:"""
```

Comment:
No longer used?

Reply:
Yes Sorry I meant to but forgot to put a comment about it, it is never used and also was throwing me off for more than 15 minutes of why is there an insert statement, so I thought better to remove this unused code.